web/src/webpack.dll.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
       '@material-ui/lab',
       '@material-ui/pickers',
       'luxon',
-      'lodash-es',
+      'lodash',
```

Comment:
`lodash` is included in the vendorPackages lower down in the list so `lodash-es` should jut be removed.